Description: A patch for autotrace.1 before 0.31.1-16
 This patch has been created to represent all changes was made direct
 to the package before implement dpkg-src 3.0(quilt) format, which has 
 a patch system.
 .
 Some changes to get a more readable manpage. 
Author: Edgar Antonio Palma de la Cruz <xbytemx@gmail.com>
Last-Update: 2011-08-07

--- autotrace-0.31.1.orig/autotrace.1
+++ autotrace-0.31.1/autotrace.1
@@ -194,7 +194,7 @@ is one of:
 .TP
 .BR epd "(Encapsulated Vectorial Graphics format)"
 .TP
-.BR eps "(Encapsulated \*(Ps)"
+.BR eps "(Encapsulated PostScript)"
 .TP
 .BR er "(Elastic Reality Shape format)"
 .TP
@@ -244,7 +244,7 @@ Print the version number of the program
 Weight factor for fitting the linewidth.
 .SH FILES
 .TP 2.2i
-/depot/bin/autotrace
+/usr/bin/autotrace
 binary
 .SH "SEE ALSO"
 gimp(1),
